mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapperConcurrencyTest.java

```diff
@@ -54,7 +55,8 @@
             CassandraBlobModule.MODULE,
             CassandraSchemaVersionModule.MODULE,
             CassandraMailboxModule.MODULE,
-            CassandraAclModule.MODULE));
+            CassandraAclModule.MODULE,
+            CassandraEventStoreModule.MODULE()));
```

Comment:
Do we need the brackets?

Reply:
Yes (scala)

Reply:
Yes, it's a scala module... Seems we do need those the way it was declared ;)